projects/components/src/table/cells/csv-generators/table-cell-boolean-csv-generator.ts

```diff
@@ -0,0 +1,13 @@
+import { TableCellCsvGeneratorBase } from '../table-cell-csv-generator-base';
+import { Injectable } from '@angular/core';
+import { isUndefined } from 'lodash-es';
+
+@Injectable({ providedIn: 'root' })
+export class TableCellBooleanCsvGenerator implements TableCellCsvGeneratorBase<boolean> {
```

Comment:
naming nit - TableCellCsvGeneratorBase -> TableCellCsvGenerator (try to avoid things like "abstract" or "base" - especially for interfaces)